fastapi_sso/sso/naver.py

```diff
@@ -25,7 +25,8 @@ async def get_discovery_document(self) -> DiscoveryDocument:
     async def openid_from_response(self, response: dict, session: Optional["httpx.AsyncClient"] = None) -> OpenID:
         return OpenID(
             id=response["response"]["id"],
-            display_name=response["response"]["nickname"],
+            email=response["response"]["email"] or None,
```

Comment:
Will the field still be there? Maybe `response["response"].get("email")` would be safer (same for the other fields), what do you say?

Reply:
Actually, I didn't know much about this part. If response["response"].get() is used, will None be automatically returned in fields that do not exist?